### PR TITLE
Minor Refining Changes

### PR DESCRIPTION
**Pull request by JordanEllis6809**
- Added helper routes for testing.
- Moved Founders Step "Add" button to the right, allowing users to [tab] through the fields and hit next.
- Fixed up styling in Founders Step.
- Refined descriptions and naming.
- Added some notes for future refactoring (data layering & generic form component).
- More type checking throughout.

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  // TODO: no "../" in imports
   // TODO: no unused imports
   // TODO: multiple imports { foo, bar } on seperate lines
```
Comment (sogasg) [inline tslint.json]: Still, do not like this enforcement :P

File: docs/contributors-guide/architecture.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+```
+
+#### 2. Lib types
+
+This is the types that are used through the project for internal communication (state, actions, exposed by integration modules etc.).
+
+#### 3. Integration types
+
+This is types that are provided by external dependencies. This types should not be accessible to the project as a whole, they should be constrained to the integrations integration module. Everything exposed by an integration module should have (2.) Lib type. This way we can easily exchange the integrations internal package.
```
Comment (cd10012) [inline docs/contributors-guide/architecture.md]: Grammer: "These are the types provided by our external dependencies"

File: src/components/DaoCreator/FeaturesStep.tsx [hunk 3]
```diff
@@ -45,8 +45,8 @@ const AddScheme: React.SFC<Props> = ({ classes, addedSchemes, actions }) => {
               Let's select the features for the DAO at creation time.
               <br />
               <br />
-              If the "Scheme Registrar" feature is added, it will be possible to
-              add and remove features at any time after the DAO is created.
+              If the "Feature Registrar" feature is added, it will be possible
```
Comment (cd10012) [inline src/components/DaoCreator/FeaturesStep.tsx]: Nit feels a lil clunky, how about: `If the "Feature Registrar" flag is turned on`

File: src/lib/integrations/daoStack/arc/votingMachines.ts [hunk 4]
```diff
@@ -26,16 +10,15 @@ export const votingMachines: VotingMachine[] = [
       {
         typeName: "ownerVote",
         valueType: "boolean",
-        displayName: "Proposer",
-        description: "The participant who called the voting mechanism",
-        defaultValue: true,
+        displayName: "Proposer Voting",
+        description: "Allow proposers to vote on their own proposals.",
```
Comment (cd10012) [inline src/lib/integrations/daoStack/arc/votingMachines.ts]: Nice catch

File: tslint.json (hunk 1 above)
Comment (JordanEllis6809) [reply tslint.json]: I definitely agree with that, will remove.

**Pull request merged**